index.html

```diff
@@ -769,12 +769,60 @@ <h3>Deactivate</h3>
 
     <section class="informative">
       <h2>Test Vectors</h2>
-      <!-- No way to do directory listing on github pages short of generating them client side :-(  Not sure if there's a better way to do this -->
-      <p>For a full list of test vectors see <a href="https://github.com/w3c-ccg/did-method-key/tree/master/test-vectors">test vectors</a>.</p>
-
+  
+      
+      <p>For a full list of test vectors see <a href="https://github.com/w3c-ccg/did-method-key/tree/main/test-vectors">test vectors</a>.</p>
+      <p>We provide only the <code>verificationMethod</code> section of the DID Document in these examples.</p>
+      
       <section class="informative">
         <h3>Ed25519 / X25519</h3>
-          <p>These DID always start with <code>z6Mk</code>.</p>
+        <p>These DID always start with <code>z6Mk</code>.</p>
+
+        <pre class="example" title="JWK Example">
+...
+"verificationMethod": [
+  {
+    "id": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp#z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp",
+    "type": "JsonWebKey2020",
+    "controller": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp",
+    "publicKeyJwk": {
+      "kty": "OKP",
+      "crv": "Ed25519",
+      "x": "O2onvM62pC1io6jQKm8Nc2UyFXcd4kOmOsBIoYtZ2ik"
+    }
+  },
+  {
+    "id": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp#z6LShs9GGnqk85isEBzzshkuVWrVKsRp24GnDuHk8QWkARMW",
+    "type": "JsonWebKey2020",
```

Comment:
```suggestion
    "type": "JsonWebKey2020",
```

How open would you be to renaming "JsonWebKey2020" to just "JsonWebKey" or "VerificationJsonWebKey" in the VC2WG? We're thinking of changing "Ed25519VerificationKey2020" to just "VerificationMultikey" or just "Multikey" as well. Specifically, for verification methods that encode their key type in the key expression, we might be able to get away with not using date-stamped versions... which would mean that we could more easily integrate them into the core https://www.w3.org/2018/credentials/v2 JSON-LD context.

The goal here would be to include a variety of public key types and cryptosuites in the core /v2 context such that you could express 80%+ of the necessary crypto-stuff using the base context. The alternative is a date-stamped slugfest wrt. what belongs in the core JSON-LD context, and I'm concerned about the amount of time we'll burn in the group doing that.

Reply:
I am strongly in support of dropping the year suffix, and the verbosity from graph node RDF Class names, but thats a discussion for the VCWG.